.changeset/clear-areas-cry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/cloudflare': patch
+'astro': patch
+---
+
+Removes an exposed internal API of the preview server `createCodegenDir`
```

Comment:
`createCodegenDir` was never documented and marked as deprecated, so I believe there's no need of a changeset. 

You might mention that we just fixed the preview server in some cases (and maybe specify the cases? )